api/URLSearchParams.json

```diff
@@ -133,32 +184,25 @@
                 "version_added": "53"
```

Comment:
It looks like this should also be version 54 (making desktop and Android line up)

```suggestion
                "version_added": "54"
```